plugins/doc_fragments/sysparm_display_value.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+    DOCUMENTATION = r"""
+options:
+  sysparm_display_value:
+    description:
+      - "Return field display values (true), actual values (false), or both (all) (default: false)."
```

Comment:
```suggestion
      - Return field display values C(true), actual values C(false), or both C(all).
```

Reply:
Add a note in the description that this parameter is added in the 2.0 release.

Reply:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment).

Reply:
Docs updated with a002c5227779f0cfed7db7f061f52240fe15fae0.

Reply:
Changelog fragment added with https://github.com/ansible-collections/servicenow.itsm/pull/188/commits/8e014069df80effb1995f3de2976d1cf5b06194e.